source/gui/settingsDialogs.py

```diff
@@ -5940,6 +5942,116 @@ def onSave(self):
 			self._providerSettings.onSave()
 
 
+class MagnifierPanel(SettingsPanel):
+	# Translators: This is the label for the magnifier settings panel.
+	title = _("Magnifier")
+	helpId = "MagnifierSettings"
+
+	def makeSettings(
+		self,
+		settingsSizer: wx.BoxSizer,
+	):
+		sHelper = guiHelper.BoxSizerHelper(
+			self,
+			sizer=settingsSizer,
+		)
+
+		# ZOOM SETTINGS
+		# Translators: The label for a setting in magnifier settings to select the default zoom level.
+		defaultZoomLabelText = _("Default &zoom level:")
+
+		zoomValues = magnifierConfig.ZoomLevel.zoom_range()
+		zoomChoices = magnifierConfig.ZoomLevel.zoom_strings()
+
+		self.defaultZoomList = sHelper.addLabeledControl(
+			defaultZoomLabelText,
+			wx.Choice,
+			choices=zoomChoices,
+		)
+		self.bindHelpEvent(
+			"magnifierDefaultZoom",
+			self.defaultZoomList,
+		)
+
+		# Store zoom values for later use
+		self.zoomValues = zoomValues
+
+		# Set default value from config
+		defaultZoom = magnifierConfig.getDefaultZoomLevel()
+		closestIndex = 0
+		minDifference = abs(self.zoomValues[0] - defaultZoom)
+		for i, value in enumerate(self.zoomValues):
+			difference = abs(value - defaultZoom)
+			if difference < minDifference:
+				minDifference = difference
+				closestIndex = i
+		self.defaultZoomList.SetSelection(closestIndex)
+
+		# FILTER SETTINGS
+		# Translators: The label for a setting in magnifier settings to select the default filter
+		defaultFilterLabelText = _("Default &filter:")
+		filterChoices = [f.displayString for f in Filter]
+		self.defaultFilterList = sHelper.addLabeledControl(
+			defaultFilterLabelText,
+			wx.Choice,
+			choices=filterChoices,
+		)
+		self.bindHelpEvent("MagnifierDefaultFilter", self.defaultFilterList)
+
+		# Set default value from config
+		defaultFilter = magnifierConfig.getDefaultFilter().displayString
+		if defaultFilter in self.defaultFilterList.GetStrings():
+			self.defaultFilterList.SetSelection(self.defaultFilterList.GetStrings().index(defaultFilter))
+		else:
+			self.defaultFilterList.SetSelection(0)
```

Comment:
Relying on display strings is fragile, this should always work

```suggestion
		defaultFilter = magnifierConfig.getDefaultFilter()
		self.defaultFilterList.SetSelection(list(Filter).index(defaultFilter))
```